Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -2075,3 +2206,20 @@ extension StatusTableViewController: PreferredGlucoseUnitObserver {
         self.refreshContext = RefreshContext.all
     }
 }
+
+extension StatusTableViewController: OnboardingDelegate {
+    func onboardingNotifying(_ notifying: OnboardingNotifying, hasNewTherapySettings therapySettings: TherapySettings) {
+        log.default("Onboarding has new therapy settings")
+
+        deviceManager.loopManager.settings.glucoseTargetRangeSchedule = therapySettings.glucoseTargetRangeSchedule
+        deviceManager.loopManager.settings.preMealTargetRange = therapySettings.preMealTargetRange
+        deviceManager.loopManager.settings.legacyWorkoutTargetRange = therapySettings.workoutTargetRange
+        deviceManager.loopManager.settings.suspendThreshold = therapySettings.suspendThreshold
+        deviceManager.loopManager.settings.maximumBolus = therapySettings.maximumBolus
+        deviceManager.loopManager.settings.maximumBasalRatePerHour = therapySettings.maximumBasalRatePerHour
+        deviceManager.loopManager.insulinSensitivitySchedule = therapySettings.insulinSensitivitySchedule
+        deviceManager.loopManager.carbRatioSchedule = therapySettings.carbRatioSchedule
+        deviceManager.loopManager.basalRateSchedule = therapySettings.basalRateSchedule
+        deviceManager.loopManager.insulinModelSettings = therapySettings.insulinModelSettings
```

Comment:
[nit] should the therapy settings just be passed to the `loopManager` (or `deviceManager`) and then it is responsible for assigning values?

Reply:
See previous comment.